specification/cat/ml_trained_models/CatTrainedModelsRequest.ts

```diff
@@ -40,7 +40,11 @@ export interface Request extends CatRequestBase {
   query_parameters: {
     allow_no_match?: boolean
     bytes?: Bytes
+    format?: string
```

Comment:
```suggestion
```

You can remove this one, as it's already defined here: https://github.com/elastic/elasticsearch-specification/blob/b3e872e566beba79bd34a8d8b54ef857e1278f1a/specification/_spec_utils/behaviors.ts#L91-L132 and you aren't updating the type.